### PR TITLE
Update LongTextType

### PR DESCRIPTION
**Pull request by archana511**

### CODE REVIEW

File: tests/monday_type_tests.py [hunk 1]
```diff
@@ -84,3 +84,74 @@ def test_should_succeed_when_to_primitive_returns_str_when_passed_in_an_int_floa
     eq_(int_value,'1')
     eq_(float_value,'1.0')
     eq_(str_value,'text')
+
+
+def test_should_succeed_when_to_native_returns_a_str_when_passing_a_longtext_value_with_api_data():
```
Comment (Ashatz) [inline tests/monday_type_tests.py:89]: Similar to before, please put the type that you are testing in the name of the test method.

Ex. test_longtext_type_should_succeed_when_...

File: tests/monday_type_tests.py [hunk 2]
```diff
@@ -84,3 +84,74 @@ def test_should_succeed_when_to_primitive_returns_str_when_passed_in_an_int_floa
...
+    longtext_value = en.cv.create_column_value(column_type, id=id, title=title, value=value)
+
+    # Act
+    longtext_type = t.LongTextType(id='longtext_column_1')
+    value = longtext_type.to_native(longtext_value)
+
+    # Assert
+    eq_(value, text)
+
+def test_should_succeed_when_to_native_returns_a_str_when_passing_an_integer_or_float_value():
```
Comment (Ashatz) [inline tests/monday_type_tests.py:106]: See above comment.

File: tests/monday_type_tests.py [hunk 3]
```diff
@@ -84,3 +84,74 @@ def test_should_succeed_when_to_primitive_returns_str_when_passed_in_an_int_floa
...
+
+    # Act
+    value_1 = longtext_type.to_native(1)
+    value_2 = longtext_type.to_native(420.69)
+
+    # Assert
+    eq_(value_1, '1')
+    eq_(value_2, '420.69')
+
+def test_should_succeed_when_to_native_returns_none_when_passing_none():
```
Comment (Ashatz) [inline tests/monday_type_tests.py:119]: See above comment.

File: tests/monday_type_tests.py [hunk 4]
```diff
@@ -84,3 +84,74 @@ def test_should_succeed_when_to_primitive_returns_str_when_passed_in_an_int_floa
...
+    longtext_type = t.LongTextType(id='longtext_column_3')
+
+    # Act
+    value = longtext_type.to_native(None)
+
+    # Assert
+    eq_(value, None)
+
+
+def test_should_succeed_when_to_primitive_returns_empty_dict_when_passing_none():
```
Comment (Ashatz) [inline tests/monday_type_tests.py:131]: See above comment.

File: tests/monday_type_tests.py [hunk 5]
```diff
@@ -84,3 +84,74 @@ def test_should_succeed_when_to_primitive_returns_str_when_passed_in_an_int_floa
...
+    longtext_type = t.LongTextType(id='longtext_column_4')
+
+    # Act
+    value = longtext_type.to_primitive(None)
+
+    # Assert
+    eq_(value, {})
+
+
+def test_should_succeed_when_to_primitive_returns_export_dict_when_passing_string_value():
```
Comment (Ashatz) [inline tests/monday_type_tests.py:143]: See above comment.

**Review by Ashatz — CHANGES_REQUESTED** [review]

File: tests/monday_type_tests.py (hunk 1 above)
Comment (archana511) [reply tests/monday_type_tests.py:89]: Ok

File: tests/monday_type_tests.py (hunk 2 above)
Comment (archana511) [reply tests/monday_type_tests.py:106]: Ok

File: tests/monday_type_tests.py (hunk 4 above)
Comment (archana511) [reply tests/monday_type_tests.py:131]: Ok

File: tests/monday_type_tests.py (hunk 5 above)
Comment (archana511) [reply tests/monday_type_tests.py:143]: Ok

**Review by Ashatz — APPROVED** [review]

**Pull request merged**